src/Page.js

```diff
@@ -822,10 +834,13 @@ class Page {
           this.collectAllPageSections();
           this.buildPageNav();
 
-          return fs.outputFileAsync(this.resultPath, htmlBeautify(
-            this.template.render(this.prepareTemplateData()),
-            Page.htmlBeautifyOptions,
-          ));
+          const renderedTemplate = this.template.render(this.prepareTemplateData());
+          const outputTemplateHTML
```

Comment:
Would 
```
const outputTemplateHTML = this.jsBeautify
  ? ...
  : ...
```
be better?

Reply:
just to clarify, do you mean bringing the "`= this.jsBeautify`" to the previous line, or to combine lines 837 and 838 to only create one `const` value as well?